platform/kube/inject/inject.go

```diff
@@ -90,8 +92,11 @@ const (
 // Defaults values for injecting istio proxy into kubernetes
 // resources.
 const (
-	DefaultSidecarProxyUID = int64(1337)
-	DefaultVerbosity       = 2
+	DefaultSidecarProxyUID   = int64(1337)
+	DefaultVerbosity         = 2
```

Comment:
Doesn't this translate to tons of logs from pilot?

Reply:
Yes, but this is no different than the kube-inject method. If desired, it would be good to change the default behavior in a follow-up PR.